src/streams/EditStreamCard.js

```diff
@@ -2,9 +2,19 @@
 import React, { PureComponent } from 'react';
 import { StyleSheet, View } from 'react-native';
 
-import { Input, Label, OptionRow, ZulipButton } from '../common';
+import type { Dispatch, StreamPostPolicy } from '../types';
+import { connect } from '../react-redux';
+import { Input, Label, OptionRow, ZulipButton, SelectionList } from '../common';
 import styles from '../styles';
 import { IconPrivate } from '../common/Icons';
+import { getIsAdmin } from '../selectors';
+import { StreamPostPolicies } from '../api/streams/streamPostPolicy';
+
+const STREAM_POST_POLICIES = {
+  [StreamPostPolicies.everyone]: { label: 'All stream members' },
+  [StreamPostPolicies.admins]: { label: 'Only organization administrators' },
+  [StreamPostPolicies.restrict_new_members]: { label: 'Only organization full members' },
+};
```

Comment:
Let's try using a real JS `Map` here, rather than an object-as-map.

One reason is that that's just cleaner in general about our intentions. Related to that is that it allows the keys to be any values we like -- in particular, they can be the actual `StreamPostPolicies.everyone` etc. -- rather than having to convert them to strings.

And then a bonus is that I suspect it will let us get rid of the need to ever convert a string *back* to a `StreamPostPolicy` value.